node/lib/util/commit.js

```diff
@@ -301,23 +362,26 @@ exports.commit = co.wrap(function *(metaRepo, all, metaStatus, message) {
     const subCommitters = Object.keys(submodules).map(commitSubmodule);
```

Comment:
The yield on arary construct is the standard way to achieve parallelism with this mechanism. I use it pretty frequently. I made the `doInParallel` method to handle situations where the normal built-in parallelism would not be able to understand and properly limit the resources we would be consuming (e.g., when forking processes).

How about a comment, e.g.:
```JavaScript
// Generate commits for submodules in parallel.
```